versions/00/oaf-00.md

```diff
@@ -249,34 +249,118 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 ```json
 {
-
+  "id": "urn:assessment:123456",
+  "name": "Student-Facing Task Statement",
+  "type": "urn:type:paper",
 }
 ```
 
-### Assignment
+### Assessment: Homework
 
 ```json
 {
-  "id": "urn:assignment:",
-  "name": "Nouns",
-  "description": "How nouns affect behavior.",
+  "id": "urn:assessment:123456",
+  "type": "urn:type:homework",
+  "name": "Student-Facing Task Statement",
+  "description": "Think about your work today, and write your best definition of area.",
+}
+```
+
+### Assessment: Paper
 
+```json
+{
+  "id": "urn:assessment:123456",
+  "type": "urn:type:paper",
+  "name": "Student-Facing Task Statement",
+  "description": "Think about your work today, and write your best definition of area.",
+  "formats": [
+    "application/msword",
+    "application/vnd.openxmlformats-officedocument.wordprocessingml.document",
+    "application/vnd.oasis.opendocument.text",
+    "application/pdf",
+  ]
 }
 ```
 
-### Question
+### Assessment: Video
 
 ```json
 {
+  "id": "urn:assessment:123456",
+  "type": "urn:type:video",
+  "name": "Student-Facing Task Statement",
+  "description": "Watch this video.",
+  "resources": [
+    {
+      "category": "video",
+      "content_type": "video/youtube",
+      "name": "",
+      "url": "https://..../"
+    }
+  ]
+}
+```
 
+### Assessment: Quiz
+
+```json
+{
+  "id": "urn:assessment:123456",
+  "type": "urn:type:quiz",
+  "name": "Student-Facing Task Statement",
+  "description": "Look for a pattern in the figures.",
+  "questions": [
+    {
+      "description": "1. How many total tiles will be in:\n\n\ta. the 4th figure?\n\tb. the 5th figure?\n\tc. the 10th figure?\n\n<img src\"\" />",
+      "ordered": [
```

Comment:
I would imagine this is optional and if it is not provided the natural order in "answers" is used.